apps/vc-api/src/vc-api/workflows/dtos/create-workflow-request.dto.ts

```diff
@@ -0,0 +1,41 @@
+import { IsString, IsObject, ValidateNested, IsOptional } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+import { Type } from 'class-transformer';
+import { WorkflowStepDto } from './workflow-step.dto';
+
+export class Steps {
+    [key: string]: WorkflowStepDto;
+}
```

Comment:
I think this is going to be a bit tricky to handle with class-validator. We'll probably need a custom validator. https://stackoverflow.com/questions/64782919/validationpipe-in-nestjs-for-a-key-value-pair-object